demo/nuxeo-date-picker/index.html

```diff
@@ -40,8 +40,8 @@
   <body unresolved>
     <dom-bind>
       <template>
-        <nuxeo-demo-section heading="Date picker" size="small">
-          <nuxeo-date-picker label="Pick a date" value="{{dateValue}}"></nuxeo-date-picker>
+        <nuxeo-demo-section heading="Date picker with 14:35:19 default time" size="small">
+          <nuxeo-date-picker label="Pick a date" value="{{dateValue}}" default-time="14:35:19"></nuxeo-date-picker>
```

Comment:
Since we currently don't have a way to set time in the controls how about calling the property just `time`?

Reply:
yes, I just tried to keep consistent with JSF UI widget naming